README.md

```diff
@@ -1,5 +1,15 @@
-# About the project
+Welcome to Rails Girsl CPH. 
+
+We are in the process of creating a platform that will help mentors and mentees find each other.  
+
+We use Ruby on Rails for our backend and React for our frontend. 
+
+If you are interested in contributing to either of these areas please follow the links above. 
+
+```Server``` will take you to the backend repository.
```

Comment:
These are not links

Reply:
What I have written is not a link, just pointing out they can use the links above to go to these repositories.